services/core/java/com/android/server/policy/PhoneWindowManager.java

```diff
@@ -4463,6 +4464,13 @@ private boolean unpinActivity(boolean checkOnly) {
     /** {@inheritDoc} */
     @Override
     public KeyEvent dispatchUnhandledKey(WindowState win, KeyEvent event, int policyFlags) {
+        boolean isDesktopInteractive =
```

Comment:
Can we move this chunk of code below line 4483 so we will also log the event (4475-4483) when debugging is turned on? This may be helpful if something goes wrong.